sources/packages/backend/libs/test-utils/src/factories/sfas-individuals.ts

```diff
@@ -0,0 +1,56 @@
+import { SFASIndividual } from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import { DataSource } from "typeorm";
+import * as faker from "faker";
+
+/**
+ * Create fake sfas individual.
+ * @param options sfas individual options.
+ * - `initialValues` sfas individual initial values.
+ * @returns created sfas individual.
+ */
+function createFakeSFASIndividual(options?: {
+  initialValues?: Partial<SFASIndividual>;
+}) {
+  const sfasIndividual = new SFASIndividual();
+  sfasIndividual.id = faker.random.number({ min: 100000000, max: 999999999 });
+  sfasIndividual.birthDate =
+    options?.initialValues.birthDate ??
```

Comment:
You're still not checking null for `initialValues`:
![image](https://github.com/bcgov/SIMS/assets/78114138/0cbb900a-344d-47ec-92b9-c31f91b2d355)